README.md

```diff
@@ -443,6 +443,15 @@ variables) that are allowed to be specified in a PR command with the
 be exported into the build environment before running the bot/build.sh script.
 
 
+```
+clone_git_repo_via = https
+```
+
+The `clone_git_repo_via` setting specifies via which mechanism the Git repository
+should be cloned. This can be either:
+* `https` (default): clone repository via HTTPS with `git clone https://github.com/<owner>/<repo>`
+* `ssh`: clone repository via SSH with `git clone git@github.com:<owner>/<repo>.git`
+
```

Comment:
Here's a suggestion about adding a little help for using 'ssh'.

````suggestion
In case of using 'ssh', one may need additional steps to ensure that the bot uses the right ssh key and does not ask for a passphrase (if the key used is protected with one). Here are a few things to consider:
- if the ssh key to be used does not have a standard name (e.g., `id_rsa`), add the following entry to `~/.ssh/config` in the bot's account
  ```
  Host github.com
    User git
    IdentityFile ~/.ssh/NAME_OF_PRIVATE_KEY_FILE
  ```
- if the key is protected by a passphrase (**highly recommended**), run an SSH agent and add the key to it
  ```
  eval $(ssh-agent -s)
  ssh-add ~/.ssh/NAME_OF_PRIVATE_KEY_FILE
  ```

````